src/source/source_cache.js

```diff
@@ -477,17 +518,8 @@ class SourceCache extends Evented {
                         covered = false;
                     }
                 } else {
-                    // Check all four actual child tiles.
-                    const children = coord.children(this._source.maxzoom);
-                    for (let j = 0; j < children.length; j++) {
-                        const childCoord = children[j];
-                        const childTile = childCoord ? this.getTile(childCoord) : null;
-                        if (!!childTile && childTile.hasData()) {
-                            retain[childCoord.id] = true;
-                        } else {
-                            covered = false;
-                        }
-                    }
+                    this._findLoadedChildren(coord, maxCoveringZoom, retain);
+                    covered = false;
```

Comment:
What does this change fix?  Would it make sense to pull this (and the corresponding changes to the source cache unit test) out into a small, separate PR?

Reply:
Ah yes – I forgot to split this out. thanks for the catch! This fixes an issue where the deleted code only checks one ZL below the current ZL so any children that are more than 1 ZL greater get released, so rapid zooming out causes a blink where child tiles are deleted before the ideal parent tiles are loaded

Reply:
👉  #5671